tcf_website/templates/base/sidebar.html

```diff
@@ -1,35 +1,49 @@
 {% load static %}
 
 <!-- Sidebar -->
-<div class="border-right
-d-flex flex-column justify-content-between" id="sidebar-wrapper">
+<div id="sidebar-wrapper" class="border-right pb-0
+     d-flex flex-column justify-content-between text-center">
   <div class="list-group list-group-flush">
     <!-- TODO: dashboard for saved courses, enrollment dates, major-specific info, etc -->
     <!-- <a href="{% url 'browse' %}"
        class="list-group-item list-group-item-action">
        <i class="fa fa-star-o fa-fw"></i>&nbsp; Dashboard
     </a> -->
     <a href="{% url 'browse' %}"
-       class="list-group-item list-group-item-action
+       class="list-group-item list-group-item-action pb-0
        {% if request.resolver_match.url_name == 'browse' %}active{% endif %}">
-       <i class="fa fa-th-large fa-fw"></i>&nbsp; Browse Courses
+       <h4><i class="fa fa-th-large fa-fw"></i></h4>
+       <p>Browse</p>
+       <hr>
     </a>
     <a href="{% url 'new_review' %}"
-       class="list-group-item list-group-item-action
+       class="list-group-item list-group-item-action pb-0
        {% if request.resolver_match.url_name == 'new_review' %}active{% endif %}">
-      <i class="fa fa-pencil-square-o fa-fw"></i>&nbsp; Write Review
+      <h4><i class="fa fa-pencil-square-o fa-fw"></i></h4>
+      <p>+Review</p>
+      <hr>
     </a>
     <a href="{% url 'about' %}"
-       class="list-group-item list-group-item-action
+       class="list-group-item list-group-item-action pb-0
        {% if request.resolver_match.url_name == 'about' %}active{% endif %}">
-      <i class="fa fa-code fa-fw"></i>&nbsp; About
+      <h4><i class="fa fa-code fa-fw"></i></h4>
+      <p>About</p>
+      <hr>
+    </a>
+    <a href="#feedback"
+       class="list-group-item list-group-item-action pb-0
+       {% if request.resolver_match.url_name == 'feedback' %}active{% endif %}">
+      <h4><i class="fa fa-thumbs-up fa-fw"></i></h4>
+      <p>Feedback</p>
+      <hr>
```

Comment:
Do we need an `hr` at the end? Because there is no `hr` at the top
```suggestion
```

Reply:
Edit: I looked at tcf1 and I see how it works, kind of like they're individual squares
except in that case maybe remove the padding on the top of the sidebar?

Reply:
Made it smaller